cdap-api/src/main/java/io/cdap/cdap/api/metrics/Metrics.java

```diff
@@ -30,6 +30,15 @@
    */
   void count(String metricName, int delta);
 
+  /**
+   * Increases the value of the specific metric by delta.
+   * @param metricName Name of the counter. Use alphanumeric characters in metric names.
+   * @param delta The value to increase by.
+   */
+  default void countLong(String metricName, long delta) {
+    this.count(metricName, (int) delta);
```

Comment:
I think it's better to add a check for the `delta` value that fits inside `int` before casting. and if it doesn't fit, throw an exception.

Reply:
I added an illegal argument exception if the value overflows int.